### ISSUE TITLE
Failed to build with rizin 0.6.3 on MacOS

### ISSUE THREAD

**Issue by zhongl** [radareorg/r2ghidra-dec#343]
Versions:

```
rizin 0.6.3 @ darwin-arm-64, package: 0.6.3 (Homebrew)
rz-pm version v0.3.3
```

Build Logs:

```
> rz-pm --debug install rz-ghidra
2024/02/13 17:46:04 Updating rz-pm-db repository...
Building rz-ghidra...
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- Configuring done (0.1s)
-- Generating done (0.0s)
-- Build files have been written to: /Users/zhongl/Library/Application Support/rz-pm/site/artifacts/rz-ghidra/0.6.0/rz-ghidra/build
[  1%] Built target pugixml
[  5%] Built target ghidra_base
[ 28%] Built target ghidra_decompiler
[ 34%] Built target ghidra_sleigh
[ 35%] Built target ghidra_libdecomp
[ 36%] Building CXX object CMakeFiles/core_ghidra.dir/src/core_ghidra.cpp.o
In file included from /Users/zhongl/Library/Application Support/rz-pm/site/artifacts/rz-ghidra/0.6.0/rz-ghidra/src/core_ghidra.cpp:9:
In file included from /Users/zhongl/Library/Application Support/rz-pm/site/artifacts/rz-ghidra/0.6.0/rz-ghidra/src/ArchMap.h:10:
In file included from /opt/homebrew/Cellar/rizin/0.6.3/include/librz/rz_core.h:7:
In file included from /opt/homebrew/Cellar/rizin/0.6.3/include/librz/rz_main.h:9:
In file included from /opt/homebrew/Cellar/rizin/0.6.3/include/librz/rz_getopt.h:4:
/opt/homebrew/Cellar/rizin/0.6.3/include/librz/rz_util.h:22:10: fatal error: 'openssl/bn.h' file not found
#include <openssl/bn.h>
         ^~~~~~~~~~~~~~
1 error generated.
make[2]: *** [CMakeFiles/core_ghidra.dir/src/core_ghidra.cpp.o] Error 1
make[1]: *** [CMakeFiles/core_ghidra.dir/all] Error 2
make: *** [all] Error 2
exit status 2
```

OpenSSL was installed on local:

```
> brew info openssl
==> openssl@3: stable 3.2.1 (bottled)
Cryptography and SSL/TLS Toolkit
https://openssl.org/
/opt/homebrew/Cellar/openssl@3/3.2.1 (6,874 files, 32MB) *
...
```

**Comment by XVilka** [comment]
Should be fixed by 0.7.0 release. Please open a new issue if this still happens.

**Comment by afjoseph** [comment]
Hey guys. I'm having the same issue. 0.7 didn't fix it. I made a PR and it works now. Check it out https://github.com/rizinorg/rz-ghidra/pull/348.
